docs/src/pages/index.tsx

```diff
@@ -0,0 +1,10 @@
+// @ts-nocheck
+import React from "react";
+
+// Webpoack alias issue with @docusarus
+// eslint-disable-next-line import/no-unresolved
+import { Redirect } from "@docusaurus/router";
```

Comment:
there's a known issue where some repo setups load this module fine, but ES & TS linters aren't able to find it. Will hopefully be fixed by Docusaurus soon